src/box/lua/net_box.c

```diff
@@ -2717,6 +2717,7 @@ luaT_netbox_transport_start(struct lua_State *L)
 		transport->self_ref = LUA_NOREF;
 		return luaT_error(L);
 	}
```

Comment:
This `FIBER_IS_SYSTEM` IMO is just an attempt to resurrect `FIBER_IS_CANCELLABLE` (which is not even fully deleted yet). I am not against it and can't propose anything better though (if we want to fix this netbox bug by making the fiber invincible). I only vote against making it a part of the public API - it truly makes the fix useless as people can easily make any fiber non-system again. I wouldn't bother even with `fiber:is_system()` and would indeed just ignore an attempt to cancel a system fiber. I trust Vova to make the further review as one closer to the core team. I can join in the end.